SmartCarMQTTApp/app/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
     defaultConfig {
         applicationId "com.example.smartcarmqttapp"
         minSdk 21
-        targetSdk 31
+        targetSdk 30
```

Comment:
Mqtt in combination with the loading screen seemed to have issues with android 12. see: https://stackoverflow.com/a/71404563